core/shared/src/main/scala/fs2/internal/CompileScope.scala

```diff
@@ -50,12 +54,23 @@ import cats.effect.Sync
  * Resources are allocated when the interpreter interprets the `Acquire` element, which is typically constructed
  * via `Stream.bracket` or `Pull.acquire`. See [[Resource]] docs for more information.
  *
- * @param id           Unique identification of the scope
- * @param parent       If empty indicates root scope. If non-emtpy, indicates parent of this scope.
+ * @param id              Unique identification of the scope
+ * @param parent          If empty indicates root scope. If non-emtpy, indicates parent of this scope.
+ * @param interruptible   If defined, allows this scope to interrupt any of its operation. Interruption
+  *                       is performed using the supplied context.
+  *                       Normally the interruption awaits next step in Algebra to be evaluated, with exception
+  *                       of Eval, that when interruption is enabled on scope will be wrapped in race,
+  *                       that eventually allows interruption while eval is evaluating.
+  *
  */
-private[internal] final class CompileScope[F[_]] private (val id: Token, private val parent: Option[CompileScope[F]])(implicit F: Sync[F]) extends Scope[F] { self =>
+private[fs2] final class RunFoldScope[F[_]] private (
```

Comment:
Looks like the rename from `RunFoldScope` to `CompileScope` was overridden in a merge conflict resolution.